jussi/cache/utils.py

```diff
@@ -31,7 +31,7 @@ def irreversible_ttl(jsonrpc_response: dict=None,
     try:
         jrpc_block_num = block_num_from_jsonrpc_response(jsonrpc_response)
         if jrpc_block_num and jrpc_block_num <= last_irreversible_block_num:
```

Comment:
Shouldn't this have something like a `DEFAULT_EXPIRE_IF_IRREVERSIBLE != -2` check? Otherwise this case always returns `TTL.DEFAULT_TTL`, which makes the default return entirely redundant.

Reply:
This case *could* be removed entirely, unless we need to keep `NO_EXPIRE` as a custom option

Reply:
I left it in case we want to have `NO_EXPIRE` as an option for irreversible blocks - currently, yes, it is redundant.

Reply:
In that case, I think we need to redefine `DEFAULT_EXPIRE_IF_IRREVERSIBLE` to equal `DEFAULT_TTL`. and instead of this case returning `TTL.NO_EXPIRE ` or `TTL.DEFAULT_TTL` it should return `DEFAULT_EXPIRE_IF_IRREVERSIBLE`

Reply:
The difference between `DEFAULT_EXPIRE_IF_IRREVERSIBLE` and simply `DEFAULT_TTL` is that it returns `TTL.NO_EXPIRE` if the block is not irreversible. So these are two different things.

Reply:
In this block we should either remove the `if` branch, or add a comment explaining why it's entirely redundant. Otherwise anyone who comes across it will be confounded:
```
        jrpc_block_num = block_num_from_jsonrpc_response(jsonrpc_response)
        if jrpc_block_num and jrpc_block_num <= last_irreversible_block_num:
            return TTL.DEFAULT_TTL
        return TTL.DEFAULT_TTL
```

Reply:
ok, updated @roadscape